pkg/logcli/client/client.go

```diff
@@ -299,6 +272,35 @@ func (c *DefaultClient) wsConnect(path, query string, quiet bool) (*websocket.Co
 		bearerToken := strings.TrimSpace(string(b))
 		h.Set("Authorization", "Bearer "+bearerToken)
 	}
+	return h, nil
+}
+
+func (c *DefaultClient) wsConnect(path, query string, quiet bool) (*websocket.Conn, error) {
+	us, err := buildURL(c.Address, path, query)
+	if err != nil {
+		return nil, err
+	}
+
+	tlsConfig, err := config.NewTLSConfig(&c.TLSConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.HasPrefix(us, "https") {
+		us = strings.Replace(us, "https", "wss", 1)
+	} else if strings.HasPrefix(us, "http") {
+		us = strings.Replace(us, "http", "ws", 1)
+	}
```

Comment:
Can't we just replace `http` with `ws` to achieve the same result?

Reply:
Good point. It was already existing code. But I fixed it now.